tensorflow/python/keras/utils/vis_utils.py

```diff
@@ -141,6 +184,9 @@ def model_to_dot(model,
 
   # Create graph nodes.
   for i, layer in enumerate(layers):
+    if i < layer_range[0] or i >= layer_range[1]:
```

Comment:
I think => might need more considering, since the end layer_range might be inclusive.

Reply:
done